src/server.js

```diff
@@ -286,10 +286,10 @@ function auth(data) {
 		}
 	} else {
 		client = manager.findClient(data.user, data.token);
-		var signedIn = data.token && data.token === client.config.token;
+		var signedIn = data.token && client && data.token === client.config.token;
```

Comment:
Seems like an odd fix. How does this entire `else` branch work if `client` is not found?